src/status_im/ui/screens/multiaccounts/key_storage/views.cljs

```diff
@@ -0,0 +1,344 @@
+                                         {:type     :secondary
+                                          :disabled (not move-keystore-checked?)
+                                          :on-press #(re-frame/dispatch next-event)
+                                          :after    :main-icons/next}
+                                         next-title]}])]]))
+
+(defn actions-not-logged-in
+  "To be used when the flow is accessed before login, will enter seed phrase next"
+  []
+  [actions-base {:next-title (i18n/label :t/enter-seed-phrase)
+                 :next-event [::multiaccounts.key-storage/enter-seed-pressed]}])
+
+(defn actions-logged-in
+  "To be used when the flow is accessed from profile, will choose storage next"
+  []
+  [actions-base {:next-title (i18n/label :t/choose-storage)
+                 :next-event [::multiaccounts.key-storage/choose-storage-pressed]}])
+
+(defview seed-phrase []
+  (letsubs
+    [{:keys [seed-word-count seed-shape-invalid?]} [:multiaccounts/key-storage]]
+    [react/keyboard-avoiding-view {:flex 1}
+     [local-topbar (i18n/label :t/enter-seed-phrase)]
+     [multiaccounts.views/seed-phrase-input
+      {:on-change-event     [::multiaccounts.key-storage/seed-phrase-input-changed]
+       :seed-word-count     seed-word-count
+       :seed-shape-invalid? seed-shape-invalid?}]
+     [react/text {:style {:color         colors/gray
+                          :font-size     14
+                          :margin-bottom 8
+                          :text-align    :center}}
+      (i18n/label :t/multiaccounts-recover-enter-phrase-text)]
+     [toolbar/toolbar {:show-border? true
+                       :right        [quo/button
+                                      {:type     :secondary
+                                       :disabled (or seed-shape-invalid?
+                                                     (nil? seed-shape-invalid?))
+                                       :on-press #(re-frame/dispatch [::multiaccounts.key-storage/choose-storage-pressed])
+                                       :after    :main-icons/next}
+                                      (i18n/label :t/choose-storage)]}]]))
+
+(defn keycard-subtitle []
+  [react/view
+   [react/text {:style {:color colors/gray}} (i18n/label :t/empty-keycard-required)]
+   [react/view {:flex-direction :row
+                :align-items    :center}
+    [react/text {:style               {:color colors/blue}
+                 :accessibility-label :learn-more
+                 :on-press #(js/alert :press)}
+     (i18n/label :learn-more)]
+    [vector-icons/icon :main-icons/tiny-external {:color  colors/blue
+                                                  :width  16
+                                                  :height 16}]]])
+
+(defn keycard-upsell-banner []
+  [react/touchable-highlight {:on-press #(.openURL ^js react/linking "https://keycard.tech/")}
+   [react/view {:background-color (if (= :dark @colors/theme) "#2C5955" "#DDF8F4")
```

Comment:
all colors must be in colors.cljs

Reply:
I talked to Gheorghe about this and he said that it's best to put it in the component itself, because nothing else depends on it: https://discord.com/channels/624205794384281629/626738033701683200/786532503379771393

Happy to move it around.